unix/man-section-7.md

```diff
@@ -0,0 +1,61 @@
+# Man Section 7
+
+For most of us, `man` pages are primarily the source of documentation for various system-level tools, be it `curl`, `ssh`, `gcc` or multitude of others.
+
+But that's not all that there is. TIL about `man` section 7 and some of its pages that you can't visit by mere accident.
+
+As `man 7 intro` puts it:
+
+```
+intro - Introduction to overview, conventions, and miscellany section
+
+Section  7  of  the  manual  provides  overviews  on  various  topics, and
+describes conventions and protocols, character set standards, the standard
+file system layout, and miscellaneous other things.
+```
+
+You can see the entire list for yourself with the following command:
+
+```
+apropos '.*' | grep '(7.* - ' | sort | less
```

Comment:
@pbrisbin The point was to provide a way to grasp the whole section, with short descriptions. Names alone *can* be descriptive, but they are usually too short (try to guess what is `raw(7)` talking about). Although I love the idea to also put the path somewhere.